src/main/java/org/zalando/nakadi/controller/CursorsController.java

```diff
@@ -201,4 +205,10 @@ public CursorsController(final CursorsService cursorsService,
         return Responses.create(Problem.valueOf(Response.Status.NOT_IMPLEMENTED, "Feature is disabled"), request);
     }
 
+    public static class CursorsAreEmptyException extends RuntimeException {
```

Comment:
Why did you decide to define this exception here?

Reply:
@lmontrieux 
I reviewed the way how underlying services treat this situation and found out, that services are not relying by any means on the amount of cursors (they may work with empty collections).
So the only reasoning for making this situation exceptional is making better user experience (which I believe should be done on a view level).
Therefore I decided to check for it there.

Reply:
@lm just updated: https://github.com/zalando/nakadi/pull/811/commits/5a556e048feb272721d0b61434e2be7f3ad98369